### PR TITLE
finish header#6

### LINKED ISSUE #6: create-and-style-for-header

### PR DESCRIPTION
**Pull request by 2H1EU8**
image
image
image

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -32,9 +32,10 @@
...
   },
   "lint-staged": {
-    "src/**/*.{html,css}": [
+    "src/**/*.{html,scss}": [
       "prettier --write --ignore-unknown",
       "eslint --fix --max-warnings=0"
     ]
-  }
+  },
+  "eslintIgnore" : ["src/**/*.scss"]
```
Comment (bao-zb) [inline package.json:40]: Why do you use it here? `eslint --fix --max-warnings=0` no longer works???

Comment (2H1EU8) [reply package.json:40]: > Why do you use it here? `eslint --fix --max-warnings=0` no longer works???

i replied in slack.

File: src/layout/header.html [hunk 2]
```diff
@@ -0,0 +1,98 @@
+<header class="main-header">
+  <div class="container">
+    <div class="main-header__top">
+      <div class="logo">
+        <p class="logo__content">H.</p>
+      </div>
+      <div class="menu">
+        <nav>
+          <ul class="menu__list">
+            <li class="header-text nav-link">Prices</li>
```
Comment (haunguyen15-tt) [inline src/layout/header.html]: Can u use the `each` tag here?

File: src/stylesheets/layout/_header.scss [hunk 3]
```diff
@@ -0,0 +1,315 @@
...
+
+        .menu__languages {
+          &__select {
+            height: pxToRem(36);
+            width: pxToRem(69);
+            display: flex;
+            background: transparent;
+            border: 2px solid #ffffff;
+            border-radius: 50px;
+            padding: 5px 5px;
```
Comment (haunguyen15-tt) [inline src/stylesheets/layout/_header.scss]: padding: 5px ?

File: src/stylesheets/layout/_header.scss [hunk 4]
```diff
@@ -0,0 +1,315 @@
...
+          margin-right: pxToRem(24.42);
+        }
+
+        .menu__languages {
+          &__select {
+            height: pxToRem(36);
+            width: pxToRem(69);
+            display: flex;
+            background: transparent;
+            border: 2px solid #ffffff;
```
Comment (haunguyen15-tt) [inline src/stylesheets/layout/_header.scss]: create constants for color

**Review by baonguyen26 — APPROVED** [review]

**Review by haunguyen15-tt — APPROVED** [review]

**Pull request merged**